app/components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import Kopfzeile from "~/components/Kopfzeile.static";
+import Logo from "~/components/Logo.static";
+import Navigation from "~/components/Navigation.static";
+import UserProfile from "~/components/UserProfile.static";
+
+export default function Header({
+  userIsLoggedIn,
+}: {
+  userIsLoggedIn: boolean;
+}) {
+  return (
+    <header>
+      <Kopfzeile />
+      {userIsLoggedIn && (
+        <div className="kern-container">
+          <div className="gap-kern-space-small flex flex-col">
+            <UserProfile />
+            <div className="flex flex-wrap items-center justify-between">
```

Comment:
I would like to suggest, that we simplify the responsive layout even a bit more. Showing up to the `xl` breakpoint a centered mobile layout and from `xl` onwards the elements as intended in Figma:

https://github.com/user-attachments/assets/8a8e73c2-3fd9-4835-b260-c465e2c7d060

To get this done we could switch to the following here:
```suggestion
            <div className="gap-kern-space-small flex flex-col items-center justify-between xl:flex-row">
```
And adjust the `ul` within the `Navigation`, will add a suggestion there as well.
```tsx
<ul className="gap-kern-space-large my-0 list-none items-center justify-between pl-0 xl:flex xl:flex-wrap">```